### PR TITLE
Landing page case list

### PR DESCRIPTION
**Pull request by eckdanny-osi**

### CODE REVIEW

File: app/javascript/_components/Caseload/Caseload.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+};
+
+const CaseloadCard = ({ status, cases, renderWaiting, renderEmpty }) => {
+  const getHeaderText = () => {
+    return status === 'ready' && cases && cases.length
+      ? `Caseload (${cases.length})`
+      : 'Caseload';
+  };
+  const renderRecords = () => {
+    if (!cases || !cases.length) {
```
[the remark this answers is not in the document]
Comment (eckdanny-osi) [reply app/javascript/_components/Caseload/Caseload.jsx]: (agreed if we're pulling in utils, lodash to be 1st stop to shop. not needed here.)

File: app/javascript/_components/Caseload/Caseload.test.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Cards } from 'react-wood-duck';
+import Caseload from './Caseload';
+
+describe('Caseload', () => {
+  it('should wrap <Card />', () => {
+    const Wrapper = shallow(<Caseload />);
+    expect(Wrapper.find(Cards).exists()).toBe(true);
+  });
```
[the remark this answers is not in the document]
Comment (eckdanny-osi) [reply app/javascript/_components/Caseload/Caseload.test.jsx]: opened #98

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
Will address Code Climate issues later, mostly false positives.

**Pull request merged**